### PR TITLE
update caption

### LINKED ISSUE #2: Activity 1: Your First Caption
Using the API, I have just created a file with a random octocat for you to caption. In this activity, we will learn how to make changes to files in our GitHub repository.

- [ ] @vvaths
- [ ] @vyas000
- [ ] @azriley
- [ ] @bwagner77
- [ ] @horvatmatthew
- [ ] @jjh121
- [ ] @klocktime
- [ ] @sinhas2020
- [ ] @tnavarra

### PR DESCRIPTION
**Pull request by vvaths**
fixes #2

### CODE REVIEW

**Review by azriley — APPROVED** [review]
ok

File: _slides/02-vvaths.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Magic new caption!
```
Comment (vyas000) [inline _slides/02-vvaths.md:6]:
```suggestion
Awesome new caption!!!
```

File: _slides/02-vvaths.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
...
```
Comment (vyas000) [inline _slides/02-vvaths.md:7]: great job!

**Review by vyas000 — CHANGES_REQUESTED** [review]

**Review by vyas000 — APPROVED** [review]
ELGTM!

**Pull request merged**